docs/notebooks/pivoted_document_length_normalisation.ipynb

```diff
@@ -0,0 +1,383 @@
4v571lK0moqpm0C24rtjCu0CytQmzRpmX53Jjkft9GDLDZfttpvLL7sUV3459Z28m+\nX8b9MUpbscW2XUiq6mHgUuBG4E76wcRdSS5P8qqm2K8Cb0zyBeC/ApfstFKSuspfspC0M8YWkiRp\nWtu2wGh1Q35Tokbb3SFW4VuS7rTA6M7+tQXG+GyB0bcs5/5umGULjLYYV2iWViG2aNOyfG7YAmN8\ntsCwBcak2ootRj0DQ5oRn1osSZI0D13pFtqVekhaHCYwpAn5YStJkhbfcCvDeelKPSQtglE/oypp\nUz4LQpIkSZJ2ky0wJEmStLBsGSnN18Y16LWn3WACQ3PV9gM9JW2nf70ZaEhaPovdDcF4SIvNZ9tp\n95jA0JwtdsChdhi47abFuuY8NyStDv8JlKRRTGBIM2bT1nEZuGkrnhuSJGnnjMsXnwkMaVcs1rfe\nkiRJi8wWfNqacfkiM4EhSZKkpeE/rjrKFnzSsjGBIUmSpCWy/N+umqSRtKpMYEgzdDTACEcDqkmX\nlSRJGmbrAkmr5wdGFUiyP8ndSe5JctkWZX42yR1JvpTkA+1XU+r/Qz84LI6dfDNSO1xekrrH2EKS\nJE1j2xYYSY4DrgReDtwL/GmSA1V110CZ04HfAP5RVT2Y5JRZVlirbvmbhUrSMjO2kKb32C9wjIUk\nrZ5RLTDOAQ5W1aGqegi4DrhwqMwbgSur6kGAqvp2+9WUJElLwthC2pHpW2cuXitWSXqsUQmMvcDh\ngddHmmmDTgeek+SPk3wmyQVtVlCSJC0VYwtpruyaKnVDP5loUnEyox7iOc4dbg/wY8B5wD7gj5Kc\ntfGtyaD19fVHx9fW1lhbWxu7opIkaTZ6vR69Xm+3NtdabGFcIUlaXMvdNX5WsUW2++mlJOcC61W1\nv3n9FuCRqrpioMy7gJur6r3N608Al1XVnw2tq/yZJ8FGlnHjydlHL9xR58fR5R5bftr1TWuremxf\n9ujfces1vJ1Jlx+nTpOsa5L3vRt1mrQ+k27j2P3PMdubZp9Mu/93YtJrZB513Eqbdeniud9VSaiq\nmURUbcUWxhXasPnnZX982nNkfrHFuHEFbHw2TXc/a/d9dfH+OuvYoo0YYLM67jRemV9sMX5MPM97\nd5v7ayfrMraY3qguJLcApyc5LckJwEXAgaEyHwXWmkqdAjwb+OpOKyZJkpaSsYUkSZrKtgmMqnoY\nuBS4EbgTuL6q7kpyeZJXNWVuBL6T5A7gk8CvVdX9M663JElaQMYWkiRpWtt2IWl1Qzb1VMMuJHYh\naaNOdiEZn11IdrYum3l2k3GFNtiFxC4kbdXJLiTjswuJXUgmtVtdSCRJkiRJkubOBIYkSZIkSeo8\nExiSJEmSJKnzTGBIkiRJkqTOM4EhSZIkSZI6zwSGJEmSJEnqPBMYkiRJkiSp80xgSJIkSZKkzjOB\nIUmSJEmSOs8EhiRJkiRJ6ryRCYwk+5PcneSeJJdtU+41SR5J8oJ2qyhJkpaJsYUkSZrGtgmMJMcB\nVwL7gTOBi5OcsUm5JwJvBj47i0pKkqTlYGwhSZKmNaoFxjnAwao6VFUPAdcBF25S7m3A24G/A9Ju\nFSVJ0hIxtpAkSVMZlcDYCxweeH2kmfaoplnn3qq6oZlU7VVPkiQtGWMLSZI0lT0j5m8bMCT5AeA/\nApcMTt5ppSRJ0tIytpAkSVMZlcC4F9g38Hof/W9KNjwR+HGglwTgqcCBJK+qqs8Pr2x9ff3R8bW1\nNdbW1qaqtCRJak+v16PX6+3W5lqLLYwrJEnqplnFFqna+ouQJHuAPwdeBnwD+BxwcVXdtUX5TwG/\nulnyIkltty2tjn5AWvS/UNs4J8Ko8+Poco8tP+36prVVPbYve/TvuPUa3s6ky49Tp0nWNcn73o06\nTVqfSbdx7P7nmO1Ns0+m3f87Mek1Mo86bqXNunTx3O+qJFTVTFo9tBVbGFdow+afl/3xac+R+cUW\n48YVsPHZNN39rN331cX766xjizZigM3quNN4ZX6xxfgx8Tzv3W3ur52sy9hiets+A6OqHgYuBW4E\n7gSur6q7klye5FU73bgkSVotxhaSJGla27bAaHVDflOihi0wbIHRRp1sgTE+W2DsbF1+S9JNxhXa\nYAsMW2C0VSdbYIzPFhi2wJjUrrTAkCRJkiRJ6gITGJIkSZIkqfNMYEiSJEmSpM4zgSFJkiRJkjrP\nBIYkSZIkSeo8ExiSJEmSJKnzTGBIkiRJkqTOM4EhSZIkSZI6zwSGJEmSJEnqPBMYkiRJkiSp80xg\nSJIkSZKkzhsrgZFkf5K7k9yT5LJN5v9KkjuSfDHJJ5Kc2n5VJUnSMjCukCRJ0xiZwEhyHHAlsB84\nE7g4yRlDxT4PvLCqngd8CHhH2xWVJEmLz7hCkiRNa5wWGOcAB6vqUFU9BFwHXDhYoKp6VfW3zcub\ngWe0W01JkrQkjCskSdJUxklg7AUOD7w+0kzbyhuAG3ZSKUmStLSMKyRJ0lT2jFGmxl1ZktcBLwB+\nebP56+vrj46vra2xtrY27qolSdKM9Ho9er3ebm3OuEKSpCU3q9giVdvHEUnOBdaran/z+i3AI1V1\nxVC5lwPvBF5aVd/eZD01altaDUnox68bfwHCGOfipuWnXd+0tqrH9mWP/h23XsPbmXT5ceo0ybom\ned+7UadJ6zPpNo7d/xyzvWn2ybT7fycmvUbmUcettFmXLp77XZWEqsroklOt27hCrdr887I/Pu05\nMr/YYty4AjY+m6a7n7X7vrp4f511bNFGDLBZHXcar8wvthg/Jp7nvbvN/bWTdRlbTG+cLiS3AKcn\nOS3JCcBFwIGhyjwfeDfwqs2CDEmSpIZxhSRJmsrIBEZVPQxcCtwI3AlcX1V3Jbk8yU81xd4BnAh8\nKMmtST46sxpLkqSFZVwhSZKmNbILSWsbsqmnGnYhsQtJG3
```

Comment:
Definitely missing description for barplots + also missed my comment about reducing number of histograms